nimbus.fml.yaml

```diff
@@ -256,6 +256,17 @@ features:
           {
             "inactive-tabs": true,
           }
+  majorReleaseOnboardingFeature:
```

Comment:
Everything else looks fine to me, but this for the simple reason that it'll cause confusion down the line.

For the MR, we have a few features that are gonna be behind the MR flag. This should be renamed to be more general to the MR, but also more specific to which MR (ie. we name most major releasese as "MajorRelease" so this would be confusing in the long run). So maybe let's re-architect it to include the year, maybe? Or, have this be an array of some sort inside that specifies years or something, if that makes sense. Actually, I'll reach out to you to chat about this. This seems like it might be more complicated that it seems for future use.